client/lib/abtest/active-tests.js

```diff
@@ -37,8 +37,8 @@ module.exports = {
 	signupStepOneCopyChanges: {
```

Comment:
Maybe we could resolve that by adding comments next to those checks. Also, if you think that this is going to be factored out in a few days anyway, feel free to keep your existing approach.